{
	"raw": {},
	"required": [
		"Filename",
		"Size"
	],
	"package": "python3-zope.i18nmessageid",
	"source": "zope.i18nmessageid (5.0.1-2)",
	"version": "5.0.1-2+b2",
	"section": "zope",
	"priority": "optional",
	"architecture": "amd64",
	"depends": [
		"python3 (<< 3.12)",
		"python3 (>= 3.11~)",
		"python3-pkg-resources",
		"python3-six",
		"python3:any",
		"libc6 (>= 2.4)"
	],
	"conflicts": [
		"zope3"
	],
	"installedSize": 67,
	"maintainer": "Debian QA Group <packages@qa.debian.org>",
	"description": "Message Identifiers for internationalization",
	"homepage": "https://pypi.python.org/pypi/zope.i18nmessageid",
	"filename": "pool/main/z/zope.i18nmessageid/python3-zope.i18nmessageid_5.0.1-2+b2_amd64.deb",
	"size": 14660,
	"md5": "097b67b0ac25c6425f775a68ceb14ab2",
	"sha256": "deb0e18153a33bfbc2d3df58732ad6c11c7551bc6a9ba66a88ff8bc18f7df901",
	"descriptionMd5": "a4ecc20de96c439e99b9eec842e6110b"
}